api/build.gradle

```diff
@@ -483,6 +479,8 @@ dependencies {
       exclude group: 'org.apache.oltu.oauth2', module: 'org.apache.oltu.oauth2.common'
     }
 
+    implementation("org.broadinstitute.dsde.workbench:leonardo-client_2.13:1.3.6-5061d24-SNAP")
```

Comment:
please update the comment at L360 to include leonardo

Reply:
added